config/sync/migrate_plus.migration.stanford_events_importer.yml

```diff
@@ -0,0 +1,163 @@
+uuid: f12377a6-f9c4-4050-8493-87df53c70bae
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: cGwmz8ZjzFcJd_ygOCdIMWZpXlLTbA3JM3CN-rniTws
+id: stanford_events_importer
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: null
+migration_group: stanford_events
+label: 'Events importer'
+source:
+  track_changes: true
+  plugin: url
+  data_fetcher_plugin: file
+  urls: null
+  data_parser_plugin: simple_xml
+  item_selector: /EventList/Event
+  fields:
+    -
+      name: guid
+      label: GUID
+      selector: guid
+    -
+      name: title
+      label: 'Event Title'
+      selector: title
+    -
+      name: body
+      label: Description
+      selector: description
+    -
+      name: event_start
+      label: 'Event start'
+      selector: isoEventDate
+    -
+      name: event_end
+      label: 'Event end'
+      selector: isoEventEndDate
+    -
+      name: link
+      label: 'Event Link'
+      selector: link
+    -
+      name: audience
+      label: 'Event Audience'
+      selector: audiences
+    -
+      name: categories
+      label: Categories
+      selector: categories
+    -
+      name: location
+      label: Location
+      selector: locationText
+    -
+      name: map_link
+      label: 'Map Link'
+      selector: mapUrl
+    -
+      name: event_sponsor
+      label: Sponsor
+      selector: sponsor
+    -
+      name: contact_email
+      label: 'Contact Email'
+      selector: contactEmail
+    -
+      name: contact_phone
+      label: 'Contact Phone'
+      selector: contactPhone
+  ids:
+    guid:
+      type: string
+  constants:
+    status: 1
+    type: stanford_event
+    stanford_html: stanford_html
+    stanford_minimal_html: stanford_minimal_html
+    map_title: 'Open in map'
+process:
+  status: constants/status
+  type: constants/type
+  title: title
+  body/value: body
+  body/format: constants/stanford_html
+  su_event_email: contact_email
+  su_event_telephone: contact_phone
+  su_event_sponsor:
+    -
+      plugin: substr
+      source: event_sponsor
+      start: 0
+      length: 120
+  su_event_location:
+    -
+      plugin: default_value
+      default_value: null
+  su_event_alt_loc: location
+  su_event_map_link/uri: map_link
+  su_event_map_link/title: constants/map_title
+  su_event_source/uri: link
+  su_event_date_time/value:
+    -
+      plugin: format_date
+      source: event_start
+      from_format: 'Y-m-d H:i:s O'
+      to_format: U
+      timezone: Etc/GMT
+  su_event_date_time/end_value:
+    -
+      plugin: format_date
+      source: event_end
+      from_format: 'Y-m-d H:i:s O'
+      to_format: U
+      timezone: Etc/GMT
+  su_event_date_time/duration:
+    -
+      plugin: stanford_events_datemath
+      operation: subtraction
+      source: event_end
+      values:
+        - event_start
+  su_event_date_time/timezone: ''
```

Comment:
is this needed? not sure what an empty string does

Reply:
We just implemented timezones on the front end. I will change this to match the feed's timezone.